src/dropdown/list/dropdown-list.component.ts

```diff
@@ -404,7 +404,7 @@ export class DropdownList implements AbstractDropdownView, AfterViewInit, OnDest
 		if (this.index < 0) {
 			this.index = 0;
 		}
-		this.getCurrentElement().focus();
+		this.getCurrentElement().focus(); 
```

Comment:
Looks like a trailing space crept in here ... 
```suggestion
		this.getCurrentElement().focus();
```